eris.go

```diff
@@ -50,7 +50,7 @@ func wrap(err error, msg string) error {
 
 	// callers(4) skips this method, Wrap(f), stack.callers, and runtime.Callers
 	stack := callers(4)
-	frame := caller(4)
+	frame := caller(3)
```

Comment:
The skip value has slightly different meanings for `callers` and `caller` so this was a bug. The wrap frames are now correct again.

Reply:
it's tested now so it'll be more difficult to break it. i'll add a comment too though